src/pages/fleerp/docs/web/reports/reports.bg.md

```diff
@@ -50,3 +50,252 @@ priority: 28
+
+![ToFuel](to-fuel-bg.png)
+
+![GroupFuel](group-fuel-bg.png)
+
+Добавена е допълнителна визуализация във вид на графика, достъпна от бутона.
+
+![ChartButton](chart-button-bg.png)
+
+![FuelChart](fuel-chart-bg.png)
+
+Възможно е скриване/показване на данни от графиката след клик върху даден етикет.
+
+![ChartOptions](chart-options-bg.png) 
+
+При посочване с мишката върху точка от графиката се визуализира прозорец с информация за дадения момент.
+
+![Popup](popup-bg.png)
+
+**Важно:** функционалноста се намира в етап на бета разработка и е възможна поява на некоретни данни, поради неправилно отчетени от датчика стойности.
+
+# Аларми
+
+Справката за аларми предоставя информация за всички възникнали аларми за даден проследяващ обект или група от обекти, в
+рамките на даден период.
+
+Предоставената справка визуализира следната информация:
+- име на алармата;
+- тип на алармата;
+- продължителност на алармата;
+- начало на алармата;
+- край на алармата;
+- допълнителна информация;
+
+![TrackingObjectAlarms](to-alarms-bg.png)
+
+![GroupAlarms](group-alarms-bg.png)
+
+Ако потребителят посочи тип на аларма, при търсенето ще се визуализират само алармите от зададения тип. В противен случай
+ще бъдат визуализирани всички възникнали аларми.
+
+# Аналогови данни
+
+Аналоговите данни предоставят визуална информация за състоянието на аналоговите портове чрез графика.
+Справката за аналогови данни се генерира за конкретен проследяван обект.
+В текущия пример се визуализират данни за скоростта, температурата и горивото на обекта.
+
+Генерирането на справката се извършва от страничния панел с избор на:
+
+- конкретен обект;
+- определен период от време;
+
+Справката съдържа данни за аналоговите портове, които са конфигурирани за дадения обект. За конкретния пример са:
+
+- **Скорост** - визуализира графично промените в скоростта;
+- **Гориво** - визуализира графично наличието на гориво;
+- **Температура** - визуализира графично каква е външната температура;
+
+За по-детайлна информация, с плъзгане на мишката по графиката се визуализират данните от аналоговите портове
+за всеки един момент от справката.
+
+![AnalogInputs](analog-inputs-bg.png)
+
+Визуализирането на данни от различни аналогови портове може да бъде скривано и показвано с един клик върху даден етикет. 
+
+![AnalogInputs](hidden-ports-bg.png)
+
+---
+
+### Важно:
+
+Възможни са шумове в аналоговите данни, които се получават от проследявания обект. Този проблем се решава
+лесно при конфигурирането на даден порт. В настройките на всеки порт има бутон "Шумове" и когато той е активиран
+графиката се изглажда *[линк](../../web/settings/tobjects/details/ports/analogs)*.
+
+![AnalogInputs](noise-false-bg.png)
+
+![AnalogInputs](noise-true-bg.png)
+
+# Цифрови данни
+
+Цифровите данни предоставят възможност на потребителя, да генерира подробна справка относно конфигурираните
+цифрови портове на даден проследяван обект или група от проследявани обекти.
+
+![Digital-inputs](digitals-bg.png)
+
+<br>
```

Comment:
I think you should remove these tags, because they create empty lines in Anchor Sidebar.

![Screenshot from 2019-07-29 10-55-25](https://user-images.githubusercontent.com/18220037/62031455-17bc6200-b1f0-11e9-97e7-9f1a8ee660ae.png)

Reply:
done